ycmd/completers/completer_utils_test.py

```diff
@@ -22,19 +22,27 @@
 from ycmd.completers import completer_utils as cu
 
 
+def _extractPatternsFromFiletypeTriggerDict( triggerDict ):
```

Comment:
Capital "E" for function name start.

Also probably a comment explaining why this function is necessary. People reading the code can't easily refer to the rationale in the PR.